server/src/middlewares/ensureAuthenticated.ts

```diff
@@ -39,7 +39,7 @@ export default function ensureAuthenticated(
     }
     const [, token] = authHeader.split(' '); // Do not use "type" of token (Bearer)
 
-    const decoded = verify(token, authConfig.jwt.secret as string);
+    const decoded = verify(token, authConfig.jwt.secret);
```

Comment:
This was throwing errors before. Is it working as-is now?

Reply:
ESLint accused that authConfig was simply already a string